docs/known-issues.md

```diff
@@ -14,3 +14,6 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | IL2CPP does not currently work on Mac as it crashes upon starting the game. This is due to a bug in Unity that does not initialize a system correctly, if one of the injected `IComponentData` contains a `long`. | 2018/08/31 | 1076596      | Use the Mono compiler.                                                | Yes    |
 | `bool` is not a [blittable type](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types), so will cause an error if used to store component data.                            | 2018/09/17 | None         | Use `BlittableBool` instead.                                          | No     |
 | Accessing component data from an entity query will segfault, therefore requests where `ResultType` is `SnapshotResultType` will be dropped before sending.                                                        | 2018/09/19 | None         | Don't send entity queries where `ResultType` is `SnapshotResultType`. | No     |
+| [Writer [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-update-component-properties): Updating a component using `Writer.Send` will lead to the SpatialOS GDK updating all properties of the component even if you indicated to only partially update a component. | 2018/09/25 | None | None | No |
```

Comment:
please format the markdown in the same way (i.e. all cells should have the same length).  it makes it easier to read in markdown

Reply:
Reformatted table. Sorry about that.